src/tests/ftest/soak/stress_48h.py

```diff
@@ -13,6 +13,13 @@ class SoakStress(SoakTestBase):
     :avocado: recursive
     """
 
+    def __init__(self, *args, **kwargs):
+        """Initialize a SoakStress object."""
+        super().__init__(*args, **kwargs)
+        soak_timeout = self.params.get(self.test_id, '/run/soak_stress/test_timeout/*')
+        self.timeout = (soak_timeout * 3600) + 1800
+        self.log.info("self.timeout: %s", self.timeout)
+
```

Comment:
We have the ability yo do this with the `timeouts` test yaml entry.
```suggestion
```

This would be implemented in the test yaml by replacing `timeout: 48H30M`:
```
timeouts:
  test_soak_stress_48h: 48H30M
  test_soak_stress_24h: 24H30M
  test_soak_stress_12h: 12H30M
```

To process the string timeout values we do need to move https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/apricot/apricot/test.py#L78-L99 to run after https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/apricot/apricot/test.py#L115.